### PR TITLE
[Bug]: Fix duartion format issue #140

### PR DESCRIPTION
**Pull request by nxanil**
@natabal @bmukheja 

Fix duration time zone format issue #140. Please have a look.
Now you may set desired time format by using `duration` property in the tooltip instead of  the `timeFormat` property. E.g - 

`{ "column": "timestamp", "label": "Value", "duration": "h [hrs], m [min], s [sec]"}`

Note: update the readme docs.
Dependent PR - https://github.com/nuagenetworks/vsd-react-ui/pull/2681

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,7 +126,8 @@ __Tolltip__ - If you want to add tooltips on an existing configuration ? Update
         // ...
         "tooltip": [
             { "column": "L7Classification", "label": "L7 Signature" },
-            { "column": "Sum of MB", "format": ",.2s"}
+            { "column": "Sum of MB", "format": ",.2s"},
+            { "column": "duration", "label": "Value", "duration": "h [hrs], m [min], s [sec]"}
```
Comment (natabal) [inline README.md]: let's call the column something else as it's confusing

File: README.md [hunk 2]
```diff
@@ -208,6 +210,16 @@ __x-axis__ __and__ __y-axis__ - (Supported Graphs - BarGraph, PieGraph, AreaGrap
 - **yLabel** y-axis title
 - **yTicks** number of ticks to use on y-axis
 - **yTickFormat** [d3 format](https://github.com/d3/d3-format) style to display y-axis labels
+- **yTickFormatType** If y axis data is in duration(miliseconds) then set `yTickFormatType: 'duration'` and define format in `yTickFormat` property to make it readable. E.g -
```
Comment (natabal) [inline README.md]: we need to add a link to valid duration formats

File: README.md (hunk 1 above)
Comment (nxanil) [reply README.md]: Actually column name is coming from database, so we can't change the name of the column . If you want then we can change the `duration` property name.

Comment (natabal) [reply README.md]: correct! But for the sake of the example let's call it something else! I don't want people to think that this column must be named 'duration'

**Pull request merged**